packages/core/admin/admin/src/content-manager/pages/ListView/utils/buildValidQueryParams.js

```diff
@@ -0,0 +1,32 @@
+import set from 'lodash/set';
+
+import createPluginsFilter from './createPluginsFilter';
+
+/**
+ * Creates a valid query params object
+ * This includes:
+ * - a filters clause
+ * - plugin options
+ */
+const buildValidQueryParams = (queryParams = {}) => {
```

Comment:
ℹ️ This is the same as buildQueryString but returning an object instead of a string. buildQueryString will eventually be removed

Reply:
Whach out for infinite loops when using an object, otherwise it's a good improvement!